README.rst

```diff
@@ -25,6 +25,8 @@ Quick start
         "OPA_DOMAIN": os.getenv("OPA_DOMAIN"),
         "OPA_VERSION": os.getenv("OPA_VERSION"),
         "IDP_URL": os.getenv("IDP_URL"),
+        "IDP_GET_USER_URL": os.getenv("IDP_GET_USER_URL"),
+        "IDP_LOGIN_URL": os.getenv("IDP_LOGIN_URL")
```

Comment:
I would say to leave only the `IDP_URL` here, and when we need the "Get User" URL for example, we can do `f"{IDP_URL}/me"`. What do you think?